### DISCUSSION TITLE
Select option templates, html, images / icons

### DISCUSSION THREAD

**Discussion by blite** [filamentphp/filament#2070] [Ideas]
It is a pretty common pattern to use templates for select dropdowns to show flags, user profile icons etc.  

See:
* https://select2.org/dropdown
* https://choices-js.github.io/Choices/ 

I personally have not researched all the pros and cons of what to do / what not to do here.  It seems logical to create a plugin for select2 or choices-js. 

I am just curious if any features like this are already on the roadmap.

**Comment by blite** [comment]
looks like this project went with choices-js, however many of the extra features are not realized yet.  Just commenting this for anyone who may search for this.

**Comment by danharrin** [comment]
#3084
